internal/cloudapi/v2/handler.go

```diff
@@ -376,21 +376,21 @@ func (h *apiHandlers) PostCompose(ctx echo.Context) error {
 					share = *gcpUploadOptions.ShareWithAccounts
 				}
 
-				// the uploaded object must have a valid extension
-				object := fmt.Sprintf("composer-api-%s.tar.gz", uuid.New().String())
+				objectName := fmt.Sprintf("composer-api-%s", uuid.New().String())
```

Comment:
Lets call this `imageName` instead, since the string is not used as is for object name